### ISSUE TITLE
MultiUploader auto submits when setAutoSubmit property is set to false

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#163]
```
What steps will reproduce the problem?
1. Create a new MultiUploader 
2. Set the auto submit property to false, multiUploader.setAutoSubmit(false);
3. Choose a new file 

What is the expected output? What do you see instead?
When the user selects a new file to queue, I don't expect for the form to 
perform a submit.

What version of the product are you using? On what operating system?
0.6.4, Windows 7 64-bit. 

Please provide any additional information below.


```

Original issue reported on code.google.com by `ndimucc...@gmail.com` on 19 Sep 2012 at 3:36

**Comment by GoogleCodeExporter** [comment]
```
This fix works for me.

Operating system:
ArchLinux
Linux 3.5-pf #1 SMP PREEMPT Mon Sep 17 11:46:22 EEST 2012 x86_64 GNU/Linux

GWT Version: 2.4.0

```

Original comment by `marius...@gmail.com` on 23 Sep 2012 at 11:34

Attachments:
- [FixAutoSubmit.diff](https://storage.googleapis.com/google-code-attachments/gwtupload/issue-163/comment-1/FixAutoSubmit.diff)

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 3 Dec 2012 at 12:18
- Changed state: **Accepted**
- Added labels: **Milestone-0.6.6**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision bd7aaaf47244.
```

Original comment by `manuel.carrasco.m` on 3 Dec 2012 at 12:20
- Changed state: **Fixed**
